styles.css

```diff
@@ -200,6 +202,382 @@ body {
+}
+
+.js-css-html-button-ul {
+  display: inline;
+}
+
+.html-css-js-button {
+  background-color: #ebebff;
+  border-radius: 8px;
+  border: 0;
+  font-family: "Poppins", sans-serif !important;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+}
+
+.html-css-js-button:hover {
+  background-color: #5a5ac9c2;
+  border-radius: 8px;
+  border: 0;
+  font-family: "Poppins", sans-serif !important;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #d5d7ebdb;
+}
+
+.footer-button {
+  border: 1px solid #396df2;
+  border-radius: 16px;
+  letter-spacing: 0.03em;
+  height: 50px;
+  font-family: "Poppins", sans-serif !important;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  color: #396df2;
+  background-color: #fff;
+}
+
+.footer-button:hover {
+  border: 2px solid #396df2;
+  border-radius: 16px;
+  letter-spacing: 0.03em;
+  height: 50px;
+  font-family: "Poppins", sans-serif !important;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  color: #396df2bc;
+  background-color: #fff;
+}
+
+.page-two-text {
+  margin: 10px 0 0 0;
+}
+
+.page-two-footer {
+  position: relative;
+  top: -15px;
+}
+
+.page-two,
+.page-three,
+.page-four,
+.page-five-div,
+.page-six-div {
+  position: absolute;
+  margin: 0 24px 0 24px;
+}
+
+.page-two {
+  top: 1500px;
+}
+
+.page-three {
+  top: 2150px;
+}
+
+.page-four {
+  top: 2800px;
+}
+
+.page-five-div {
+  top: 3450px;
+  height: 1200px;
+  border-radius: 0 60px 0 0;
+}
+
+.about-myself {
+  margin: 114px 0 0 0;
+  height: 101px;
+}
+
+.connect-icons {
+  margin: 20px 0 18px 0;
+  display: flex;
+  width: 150px;
+  height: 50px;
+}
+
+.lang {
+  margin: 0 0 0 10px;
+  font-family: "Poppins", sans-serif !important;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+}
+
+.lang-list-options,
+.skill-frameworks-options {
+  display: flex;
+  position: relative;
+  top: -20px;
+  left: 250px;
+  border: 1px solid #35355312;
+  border-radius: 50%;
+}
+
+.lang-list-div,
+.footer-div {
+  background: #f7f7f9;
+  border-radius: 8px;
+  margin: 0 20px 5px 10px;
+  height: 66.72px;
+}
+
+ul {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.html-css-round-button {
+  border: 0;
+  background-color: #f7f7f9;
+  font-family: "Poppins", sans-serif !important;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #253858;
+  align-items: center;
+}
+
+.skill-frameworks-options {
+  position: relative;
+  top: -30px;
+}
+
+.page-five-footer-div {
+  border-radius: 0;
+  padding: 10px 0 0 0;
+  border-bottom: 1px solid #dfe1e6;
+  background-color: #fff;
+  font-family: "Poppins", sans-serif !important;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  color: #000;
+}
+
```

Comment:
- Kindly make sure to stick to the provided design by adjusting your margins and padding on the `about myself` section. 😉 

  - ### Your design
  ![y-w-about](https://user-images.githubusercontent.com/87197412/185703676-d60223b8-c01b-4b7c-aee9-4a83ceb46de2.PNG)


  - ### Figma design
  ![r-w-about](https://user-images.githubusercontent.com/87197412/185703684-82564c62-da41-415a-bd70-25a204fb0f3f.PNG)